packages/react-composites/tests/browser/call/hermetic/fixture.ts

```diff
@@ -95,6 +95,52 @@ export function defaultMockCallAdapterState(participants?: MockRemoteParticipant
   };
 }
 
+/**
+ * Create the default {@link MockCallAdapterState}for call readiness hermetic e2e tests.
+ */
+export function defaultMockCallAdapterStateDeviceDisabled(
+  participants?: MockRemoteParticipantState[]
+): MockCallAdapterState {
+  const remoteParticipants: Record<string, MockRemoteParticipantState> = {};
+  participants?.forEach((p) => {
+    remoteParticipants[toFlatCommunicationIdentifier(p.identifier)] = p;
+  });
+  return {
+    displayName: 'Agnes Thompson',
+    isLocalPreviewMicrophoneEnabled: false,
+    page: 'configuration',
+    userId: { kind: 'communicationUser', communicationUserId: '1' },
+    devices: {
+      isSpeakerSelectionAvailable: true,
+      cameras: [],
+      microphones: [],
+      speakers: [
+        {
+          name: '',
+          id: 'speaker:',
+          isSystemDefault: true,
+          deviceType: 'Speaker'
+        }
+      ],
+      unparentedViews: [],
+      selectedMicrophone: {
+        name: '',
```

Comment:
does this match the state of a actually disabled mic? I would have thought this object would just be undefined?